blocks/callout/callout.css

```diff
@@ -45,15 +45,14 @@
     @container callout-wrapper (min-width: 49rem) {
       margin-block-start: 0;
     }
+    @media (min-width: 48rem) {
+      width: unset;
+    }
```

Comment:
but is button-sized based on screen, not container

Reply:
Yeah - it'd be cleaner if we just defined that in `styles.css` as a rule for all buttons. That way we have one less nested styling definition in some block CSS module.